.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+Contributing to the Sustain OSS Design Inventory
+================================================
+
+Thanks for your interest in working on the Sustain OSS Design Inventory!
+We appreciate your interest and enthusiasm.
+This document explains the contribution process, the different components of the project, and how to create a development environment to work on the site.
+
+
+**Table of contents**:
+
+1. [Contribution process](#process)
+1. [Structure and components](#components)
+1. [How to create a development environment](#dev-env)
+
+
+## <a id="process"></a>Contribution process
+
+The Sustain OSS Design Inventory is governed by the [Sustain OSS Design & UX Working Group](https://sustainoss.org/working-groups/design-and-ux/).
+The Working Group is part of the [Sustain OSS community](https://sustainoss.org/about/).
+Decisions about the project, the content published herein, and other aspects are generally headed up by these fine folks (in alphabetical order):
+
+1. Eriol Fox
+1. Georgia Bullen
+1. Justin W. Flory
+1. Memo Esparza
+1. Richard Littauer
+
+Since the team is small and our efforts are still getting started, the governance of this project is fluid and may change over time to help us grow and scale the effort, as needed.
+
+### Governance model: Lazy consensus
+
+When reviewing changes or new ideas to this project, the maintainers will adhere to a **lazy consensus** model of decision-making.
+
+**What does lazy consensus mean?**
+In order for a change or new effort to move forward, it must receive _at least one_ "+1" vote from a maintainer, and no "-1" votes from any maintainer.
+This enables work to move forward quickly if there are no major disagreements.
+For minor and non-controversial changes, a single approval allows a change to move forward with little resistance.
+If there is a change or new idea a core maintainer raises issue with, the "-1" vote requires the process to pause and have a longer discussion around a change or new idea.
+
+**What constitutes a vote?**
+In the context of this repository, a vote can mean a review on a Pull Request or a comment on an Issue.
+A maintainer can give a "+1" vote or approve a Pull Request if they agree with the change.
```

Comment:
I don't think this is our current process. Perhaps something like: "A maintainer can give a general approval by saying something like "+1" or "LGTM", or just approve a PR using GitHub's review feature." Otherwise, this seems dogmatic.

Reply:
@RichardLitt My intent was not to capture our current process (do we have one?). I was setting out to document a norm that we could agree on following as a group. Although I'm not sure I see the difference between what I've written here and your proposed edit, unless you are suggesting to strike this entire process?

Reply:
I hear you. Your proposal gives off the impression that the only way to vote is to say "+1" or "-1", when those are personal language choices. If someone just wrote "I like this", that would also work for our current process, but the writing here is too restrictive for that use-case. I propose we broaden your writing to include language choices.

Reply:
Ah! Yeah, that makes sense and it is fair feedback. I can amend this. I'll probably get to this sometime next week and include any other feedback from other reviewers at the same time.